app/components/blacklight/document/bookmark_component.html.erb

```diff
@@ -6,7 +6,8 @@
              data: {
                present: t('blacklight.search.bookmarks.present'),
                absent: t('blacklight.search.bookmarks.absent'),
-               inprogress: t('blacklight.search.bookmarks.inprogress')
+               inprogress: t('blacklight.search.bookmarks.inprogress'),
+               doc_id: @document.id
```

Comment:
Looks good, but let's go with `document_id` as that's what's more commonly used throughout the codebase.